common/models/templates/test/test-trie-traversal.js

```diff
@@ -101,6 +105,38 @@ describe('Trie traversal abstractions', function() {
     assert.isEmpty(eKeys);
   });
 
+  it('direct traversal with simple internal nodes', function() {
```

Comment:
The description is not very fluent -- ideally, should be something that works with "it", like:

```suggestion
  it('directly traverses a trie model with simple internal nodes', function() {
```

But this is a nit only, I don't really care that much